components/raftstore/src/store/fsm/apply.rs

```diff
@@ -178,6 +179,16 @@ where
     }
 }
 
+impl<S> HeapSize for PendingCmdQueue<S>
+where
+    S: Snapshot,
+{
+    #[inline]
+    fn heap_size(&self) -> usize {
+        self.normals.capacity() * mem::size_of::<PendingCmd<S>>()
```

Comment:
I think it's better to sum of all fields' heap size instead of calculating here.

Reply:
I think we can write `self.normals.capacity() * (mem::size_of::<PendingCmd<S>>() + 8 /*callback fn point*/)`.

Reply:
Function pointer is not at the heap.